### PR TITLE
 MusicAlbum schema

### PR DESCRIPTION
**Pull request by VicPeralta**
# Schema for musicAlbum and genre

This PR includes the schema file to create the tables for musicAlbum and genre

new file:   schema.sql

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- Create table genre
+DROP TABLE IF EXISTS "genre";
+CREATE TABLE "genre"(
```
Comment (richardoppiyo) [inline schema.sql]: It's better to join the commands together:

CREATE TABLE genre (
    id integer NOT NULL,
    name varchar(255) NOT NULL,
   
    PRIMARY KEY (id)
);

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "id" INTEGER NOT NULL,
+    "name" VARCHAR(255) NOT NULL
+);
+ALTER TABLE
+    "genre" ADD PRIMARY KEY("id");
+
+-- Create table music_album
+DROP TABLE IF EXISTS  "music_album";
+CREATE TABLE "music_album"(
+    "id" INTEGER NOT NULL,
```
Comment (richardoppiyo) [inline schema.sql]: CREATE TABLE  music_album(
     id  INTEGER NOT NULL,
     genre  INTEGER NOT NULL FOREIGN KEY REFERENCES  genre(id),
     publish_date  DATE NOT NULL,
     archived BOOLEAN NOT NULL,
     on_spotify BOOLEAN NOT NULL
 
   PRIMARY KEY(id)
 );

**Review by richardoppiyo — CHANGES_REQUESTED** [review]
Hello @VicPeralta  👋 ,

Good job on creating the sql table structure 👏.
However, it could be better if you create tables with the constraints altogether rather than altering the tables after creation. This would reduce the size of the code as well as execution time. Check the comment.

Also, remember to remove the  ""  form the table column names.

File: schema.sql (hunk 1 above)
Comment (VicPeralta) [reply schema.sql]: Hi @richardoppiyo thanks for your comment. I update the query to include the constraint in the same statement:
CREATE TABLE genre(
    id INTEGER NOT NULL PRIMARY KEY,
    name VARCHAR(255) NOT NULL
);

File: schema.sql (hunk 2 above)
Comment (VicPeralta) [reply schema.sql]: Hi @richardoppiyo ,
I updated the statement to include the constraint in the same create statement:
CREATE TABLE music_album(
    id INTEGER NOT NULL PRIMARY KEY,
    genre INTEGER NOT NULL,
    publish_date DATE NOT NULL,
    archived BOOLEAN NOT NULL,
    on_spotify BOOLEAN NOT NULL,
    FOREIGN KEY (genre)
        REFERENCES genre (id)
);

**Review by richardoppiyo — APPROVED** [review]
Hi @VicPeralta 

Congratulations for the work well done!

**Pull request merged**